cmd/launcher/flare.go

```diff
@@ -28,32 +30,16 @@ func runFlare(args []string) error {
 	launcher.SetDefaultPaths()
 
 	var (
-		flagset = flag.NewFlagSet("flare", flag.ExitOnError)
-		flSave  = flagset.String(
-			"save",
-			"local",
-			"local | upload",
-		)
-		flOutputDir = flagset.String(
-			"output_dir",
-			".",
-			"path to directory to save flare output",
-		)
-		flUploadRequestURL = flagset.String(
-			"upload_request_url",
-			"",
-			"URL to request a signed upload URL",
-		)
+		flagset            = flag.NewFlagSet("flare", flag.ExitOnError)
+		flSave             = flagset.String("save", "upload", "local | upload")
+		flOutputDir        = flagset.String("output_dir", ".", "path to directory to save flare output")
+		flUploadRequestURL = flagset.String("upload_request_url", "", "URL to request a signed upload URL")
```

Comment:
should we populate this with the default URL or do we want to send down from control server? But sending from control server would mean that a non-enrolled launcher would still need this as a cmd line flag.

Reply:
Oh! I hadn't even realized it didn't default to something. Yeah, let me update that. Thank you

Reply:
How's that?